core-libs/setup/ssr/optimized-engine/ssr-optimization-options.ts

```diff
@@ -124,6 +125,14 @@ export interface SsrOptimizationOptions {
    * By default, the logger is disabled, meaning that logs in SSR are not enhanced.
    */
   logger?: true | ExpressServerLogger; //CXSPA-3680 - allow only providing ExpressServerLogger implementations
+
+  /**
+   * Config for transforming logs.
+   * It allows to add, remove or replace log transformers.
+   * The default transformers are: TraceparentTransformer
+   * @see ExpressLogTransformer
+   */
+  logTransformers?: ExpressLogTransformerConfig;
```

Comment:
I suggest to use `deepMerge() from @spartacus/core` for merging custom config into the default config. And the `logTransformers` should have type simply simply `TransformerConfig` (`LogTransformerConfig`)